pyproject.toml

```diff
@@ -52,3 +44,14 @@ disable = """
     raise-missing-from,
     too-many-instance-attributes,
 """
+
+[tool.ruff]
+line-length = 79
+target-version = "py313"
```

Comment:
Any reason for python 3.13 instead of 3.12?

Reply:
Thank you for the question,

I'm currently using python 3.13 in my environment runtime, thought it would be better to keep the target high already.

The API spec says it requires python 3.12 or above, the base image is currently at 3.12 but the policy here is to always keep it as high as possible.

Differently from the libdev context where people might be using the lib in older python versions, we don't have to maintain any legacy support, just be sure every test passes with each python release .

Reply:
just asked bc python 3.13 is really recent (1 month).

Reply:
patched https://github.com/RocketPy-Team/Infinity-API/pull/37/commits/3cfd570d222f0220be660cbaa0d62e4e96f25aea to keep ruff python version rule the same as base image version